README.md

```diff
@@ -90,7 +90,7 @@ docker-compose up
 
 
 #### Customizing the Default Application Configuration
-The default application settings (database url, http port, etc.) can be changed by updating the [config file] (https://github.com/OWASP/NodejsGoat/blob/master/config/env/all.js).
+The default application settings (database url, http port, etc.) can be changed by updating the [config file] (https://github.com/OWASP/NodeGoat/blob/master/config/env/all.js).
```

Comment:
Well, I had changed it for my fork to binarymist, but then realised that NodejsGoat was no longer correct.